syntax = "proto3";

message SensorData {
    string sensor_id = 1;
    double temperature = 2;
    double humidity = 3;
    double pressure = 4;
    int64 timestamp = 5;
}